food-app/src/pages/ContactPage.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+export const ContactPage = () => {
+  const [sent, setSent] = useState(false);
+
+  return (
+    <section className="container px-4 mx-auto">
+      <div className="flex flex-col">
+        <h1 className="text-3xl font-extrabold mb-5">Envianos un mensaje</h1>
+        {sent ? (
+          <div>Tu mensaje fue enviado, te responderemos pronto.</div>
```

Comment:
```suggestion
          <div role="alert">Tu mensaje fue enviado, te responderemos pronto.</div>
```

> Sometimes, for example, when using AJAX techniques, the browser is not loading a new page but shows changes, such as form errors, dynamically on the page. The list of errors should be inserted into a prominent container on the top to inform the user in such a case. In addition to the advice above, this container should have the role attribute set to alert to make assistive technology users aware of this change.

taken from [this W3C page in the Listing Errors section](https://www.w3.org/WAI/tutorials/forms/notifications/)